sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py

```diff
@@ -248,9 +252,13 @@ def test_crud_operations(self, vault_client, **kwargs):
         self.assertEqual(cert.id, cert_bundle.id)
         self.assertNotEqual(cert.properties.updated_on, cert_bundle.properties.updated_on)
 
+        polling_interval = 0 if self.is_playback() else 2
```

Comment:
Is there a reason not to test the client method's default interval, i.e. use `None` in live tests instead of 2?

Reply:
changed that, thanks!